services/spire/federation/clusterfederatedtrustdomain-cluster1.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: spire.spiffe.io/v1alpha1
+kind: ClusterFederatedTrustDomain
+metadata:
+  name: cluster2.demo
+spec:
+  trustDomain: cluster2.demo
+  bundleEndpointURL: https://172.16.17.3:8443 #replace with ip of spire server endpoint on cluster 2
```

Comment:
Again mask this value as bundleEndpointURL: <Replace with IP of sprire server, e.g. https://172.16.17.3:8443>